<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startQuiz</class>
 <widget class="QDialog" name="startQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>703</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ImagesAndIcons.qrc">:/images/icons &amp; images/start quiz page.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 32px;
    font-weight: bold;
    color: cyan;
    text-align: center;
    text-shadow: 0px 0px 10px cyan;
}</string>
   </property>
   <property name="text">
    <string>üöÄ Gear Up for Quiz Fun!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string> Choose your quest, set your pace, and conquer! ‚ú®</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    color: white;
    text-shadow: 0px 0px 5px white;
}</string>
   </property>
   <property name="text">
    <string>üåü Choose Your Challenge Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    color: white;
    text-shadow: 0px 0px 5px white;
}</string>
   </property>
   <property name="text">
    <string>üìö What's Your Topic?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    color: white;
    text-shadow: 0px 0px 5px white;
}</string>
   </property>
   <property name="text">
    <string>üéØ Points System:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEasy">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    font-size: 18px;
    color: #00ff00;
    padding: 5px;
    border: none;
    spacing: 10px;
}
QRadioButton::indicator {
    width: 15px;
    height: 15px;
}
QRadioButton::indicator::unchecked {
    border: 2px solid #00ff00;
    background: transparent;
}
QRadioButton::indicator::checked {
    border: 2px solid #00ff00;
    background: #00ff00;
}</string>
   </property>
   <property name="text">
    <string>Easy:  üå± Easy Peasy</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonMedium">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    font-size: 18px;
    color: #00ff00;
    padding: 5px;
    border: none;
    spacing: 10px;
}
QRadioButton::indicator {
    width: 15px;
    height: 15px;
}
QRadioButton::indicator::unchecked {
    border: 2px solid #00ff00;
    background: transparent;
}
QRadioButton::indicator::checked {
    border: 2px solid #00ff00;
    background: #00ff00;
}</string>
   </property>
   <property name="text">
    <string>Medium:  ‚öñÔ∏è Let‚Äôs Think!</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonHard">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    font-size: 18px;
    color: #00ff00;
    padding: 5px;
    border: none;
    spacing: 10px;
}
QRadioButton::indicator {
    width: 15px;
    height: 15px;
}
QRadioButton::indicator::unchecked {
    border: 2px solid #00ff00;
    background: transparent;
}
QRadioButton::indicator::checked {
    border: 2px solid #00ff00;
    background: #00ff00;
}</string>
   </property>
   <property name="text">
    <string>Hard:  üî• Bring It On!</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>280</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    font-size: 16px;
    color: white;
    background-color: #222;
    border: 2px solid #ff00ff;
    border-radius: 5px;
    padding: 5px;
}
QComboBox QAbstractItemView {
    background-color: #111;
    border: 1px solid #ff00ff;
    color: white;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Programming Languages</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data Structures &amp; Algorithms (DSA)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object-Oriented Programming (OOP)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>250</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>üîç Explore Categories Below:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #ffff00;
    text-shadow: 0px 0px 5px #ffff00;
}</string>
   </property>
   <property name="text">
    <string>Easy: 10 points/question
Medium: 20 points/question
Hard: 30 points/question</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/dell/Downloads/arrow_14034415.png</normaloff>C:/Users/dell/Downloads/arrow_14034415.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    color: white;
    background-color: #ff00ff;
    border: 2px solid #00ffff;
    border-radius: 10px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #00ffff;
    color: black;
}
QPushButton:pressed {
    background-color: #ff00ff;
    border: 2px solid white;
    color: white;
}</string>
   </property>
   <property name="text">
    <string> Ready to Play!</string>
   </property>
   <property name="icon">
    <iconset resource="ImagesAndIcons.qrc">
     <normaloff>:/images/icons &amp; images/play now icon.png</normaloff>:/images/icons &amp; images/play now icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/dell/Downloads/are you ready image.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>radioButtonEasy</zorder>
  <zorder>radioButtonMedium</zorder>
  <zorder>radioButtonHard</zorder>
  <zorder>categoryComboBox</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_9</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="ImagesAndIcons.qrc"/>
 </resources>
 <connections/>
</ui>
